.github/workflows/acceptance.yml

```diff
@@ -308,7 +308,7 @@ jobs:
     needs: [acceptance, frontend, chartcuterie]
     name: triggers visual snapshot
     # Do not execute on forks or on master
-    if: github.repository == 'getsentry/sentry' && github.ref != 'refs/heads/master'
+    if: github.head_repository.full_name == 'getsentry/sentry' && github.ref != 'refs/heads/master'
```

Comment:
This means that `visual-diff` in _this_ workflow won't execute when the PR was opened from a fork while
`visual-diff` with `workflow_run` will execute.